openedx/frontend-app-learner-dashboard
# fix: display SUPPORT_URL only if is configured

**Description**

This PR removes the link of `SUPPORT_URL` from the Header if the service is not configured. 

It aims to solve the following issue https://github.com/openedx/frontend-app-learner-dashboard/issues/373 

**How to test**

1. Create a instance with `frontend-app-learner-dashobord` mounted in the current branch.
2. Launch the instance and open it in the browser.
3. Access the learner dashboard.
4. By default you should not see help nav button. 
6. Set the `SUPPORT_URL` config variable.
7. Now the tab should be available.
